CONTRIBUTING.md

```diff
@@ -63,34 +63,24 @@ integration tests will create local registries on ports 5000 and 6000.
 
 ## Configuring Eclipse
 
-Jib is a mix of Gradle and Maven projects.  But since we do not
-publish all artifacts, we instead have the projects reference
-source from their dependencies directly using `sourceSets` and
-`build-helper-maven-plugin`.  This complicates importing these projects
-as the Gradle and Maven tooling for Eclipse faithfully replicate the
-direct source references.  This has a side-effect that classes
-appear both in the originating project _and_ in the subprojects,
-and can interfere with Eclipse's incremental compilation.
+Jib is a mix of Gradle and Maven projects, however we build everything using one
+unifed gradle build. There is special code to include some projects directly as
+source, but importing your project should be pretty straight forward.
 
-The following instructions first import the projects, and then replaces
-the source-set-style references to use Eclipse's project references
-instead.
+// TODO: Check with chanseok/brian on how importing into eclipse will work
 
   1. Ensure you have installed the Gradle tooling for Eclipse, called
      _Buildship_ (available from [the Eclipse
      Marketplace](https://marketplace.eclipse.org/content/buildship-gradle-integration)).
-  2. **Import the Maven project:** Use _File &rarr; Open Projects from File System..._
```

Comment:
I know it renders correctly, but since you deleted step 2 the numbering in the plain text is off.